saleor/graphql/core/utils.py

```diff
@@ -0,0 +1,6 @@
+def handle_seo_fields(data):
```

Comment:
Maybe `clean_seo_fields`? I know it is not really cleaning but it's always used as part of "clean".